tests/test_cli.py

```diff
@@ -84,6 +84,33 @@ def test_compress_already_compressed_force():
     assert h1 != h2
 
 
+def test_compress_correct_offset():
+    """Testing if correction of offset works in compress-function"""
+    # Arrange
+    path_in = retrieve_data("rtdc_data_hdf5_rtfdc.zip")
+    # Compress with and without offset correction
+    path_out = path_in.with_name("compressed.rtdc")
+    path_out_nc = path_in.with_name("compressed_nc.rtdc")
+
+    with new_dataset(path_in) as ds:
+        fl_max_old = ds["fl1_max"]
+        fl_offset = min(np.min(ds["trace"]["fl1_raw"]), 1)
+
+    # Act
+    cli.compress(path_out=path_out, path_in=path_in, correct_offset=True)
+    cli.compress(path_out=path_out_nc, path_in=path_in, correct_offset=False)
+
+    # Assert
+    with new_dataset(path_out) as ds:
+        assert "baseline 1 offset" in ds.config["fluorescence"]
+        assert fl_offset == ds.config["fluorescence"]["baseline 1 offset"]
+        assert (ds["fl1_max"] == fl_max_old - (fl_offset-1)).all()
```

Comment:
The `-1` should not be here (see my comments in correct.py). Unless I misunderstood something...